vimeo-networking/src/main/java/com/vimeo/networking/model/VideoFile.java

```diff
@@ -63,18 +64,12 @@ public String toString() {
         }
     }
 
-    @Deprecated
-    @UseStag
     public enum VideoQuality {
```

Comment:
Totally can be. We have a ticket just for that though, and since I didn't introduce any new fields, I'd rather wait for that ticket.